### PR TITLE
New michal bassa

### PR DESCRIPTION
**Pull request by michalBassa**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import index from 'index.html'
-const size=5;
+const sum=5;
+const count=10;
 
 console.log(size);
-
+console.log(sum*10);
```
Comment (yehuditKritenberger) [inline index.js]: ok

**Review by yehuditKritenberger — CHANGES_REQUESTED** [review]

File: index.js [hunk 2]
```diff
@@ -11,12 +12,13 @@ console.log("Michal Bassa");
 
 console.log("conflict");
 
-const array=[1, 2, 3];
+const array=[1, 2, 3,4, 5 ];
```
Comment (yehuditKritenberger) [inline index.js]: ok

**Review by yehuditKritenberger — CHANGES_REQUESTED** [review]

File: index.js [hunk 3]
```diff
@@ -1,8 +1,9 @@
 import index from 'index.html'
-const size=5;
+const sum=5;
```
Comment (yehuditKritenberger) [inline index.js]: change parameter

**Review by yehuditKritenberger — CHANGES_REQUESTED** [review]

File: index.js (hunk 2 above)
Comment (michalBassa) [reply index.js]: what mor good?

**Pull request closed without merging**